src/IISIntegration/src/Microsoft.AspNetCore.Server.IISIntegration/Microsoft.AspNetCore.Server.IISIntegration.csproj

```diff
@@ -21,7 +21,6 @@
     <PackageReference Include="Microsoft.AspNetCore.Http" Version="$(MicrosoftAspNetCoreHttpPackageVersion)" />
     <PackageReference Include="Microsoft.AspNetCore.Http.Extensions" Version="$(MicrosoftAspNetCoreHttpExtensionsPackageVersion)" />
     <PackageReference Include="Microsoft.AspNetCore.HttpOverrides" Version="$(MicrosoftAspNetCoreHttpOverridesPackageVersion)" />
-    <PackageReference Include="Microsoft.AspNetCore.HttpSys.Sources" PrivateAssets="All" Version="$(MicrosoftAspNetCoreHttpSysSourcesPackageVersion)" />
```

Comment:
It looks like this is causing compiler errors. I recommend we 
1. moving the folder Microsoft.AspNetCore.HttpSys.Sources of source code to `$repoRoot/src/Shared/HttpSys/`
2. Add a file in $repoRoot/src/Shared/HttpSys/Directory.Build.props like this: https://github.com/aspnet/AspNetCore/blob/release/2.1/src/Shared/Diagnostics/Directory.Build.props since I don't believe this source is used outside of aspnet/AspNetCore
3. Update IIS and HttpSys servers to `<Compile Include="$(SharedSourceRoot)HttpSys\**\*.cs" />`

Reply:
Why not have a top level Directory.Build.props for all shared folders?

Reply:
I like this idea. I forgot that this commit was only in master: https://github.com/aspnet/AspNetCore/commit/7ffbb6ed3b6d0b2eee33c59f2d791b47794e3c92. You can probably cherry-pick some of it. Once we finish #4246, I expect we will only have one or two .Sources packages, so `IsPackable=false` is a better default.